### PR TITLE
Report DB metrics

### PR DESCRIPTION
**Pull request by arijitAD**
https://github.com/vulcanize/go-ethereum/issues/138

### CODE REVIEW

**Review by i-norden — APPROVED** [review]

**Review by i-norden — APPROVED** [review]
LGTM. Has this been tested?

File: statediff/service.go [hunk 1]
```diff
@@ -169,6 +171,10 @@ func New(stack *node.Node, ethServ *eth.Ethereum, cfg *ethconfig.Config, params
 		if err != nil {
 			return err
 		}
+
+		if metrics.Enabled {
```
Comment (i-norden) [inline statediff/service.go]: ReportDBMetrics checks if metrics.Enabled is true and exits if not, so we don't need to check here.

**Review by i-norden — CHANGES_REQUESTED** [review]

File: statediff/service.go (hunk 1 above)
Comment (arijitAD) [reply statediff/service.go]: Done

Comment (i-norden) [reply statediff/service.go]: Thanks! Have you had a chance to run/test with this enabled?

Comment (arijitAD) [reply statediff/service.go]: Screenshot 2021-10-26 at 4 53 39 PM
Verified in Prometheus

**Pull request merged**